spec/services/find_availability_spec.rb

```diff
@@ -0,0 +1,55 @@
+describe FindAvailability, type: :service do
+  let!(:now) { Date.new(2019, 9, 19) }
+
+  before { travel_to(now) }
+  after { travel_back }
+
+  it "without not online motif" do
```

Comment:
without not online ce n'est pas tres clair. 
si tu veux tester ce qui se passe quand le motif n'est pas `online`, je pense qu'il faut que tu crées la plage d'ouverture avant. s'il n'y a pas de plage d'ouverture, qu'il soit online ou pas il n'y aura aucun créneau je crois